app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -82,11 +82,47 @@ export class ReleaseNotes extends React.Component<IReleaseNotesProps, {}> {
     )
   }
 
+  /**
+   * If there is just one release, it returns it. If multiple, it merges the release notes.
+   */
+  private getDisplayRelease = () => {
+    if (this.props.newReleases.length === 1) {
```

Comment:
With how often we access `this.props.newReleases` in this method I think we should probably destructure it from `this.props` to clean things up a little